nevergrad/optimization/optimizerlib.py

```diff
@@ -1029,12 +1030,17 @@ def __init__(
             assert self.budget is not None, "Either scale or budget must be known in _Rescaled."
             scale = math.sqrt(math.log(self.budget) / self.dimension)
         self.scale = scale
+        self.initialization = (
+            np.zeros(self.dimension)
+            if translate is None
+            else self.parametrization.random_state.normal(0.0, translate, size=self.dimension)
+        )
         assert self.scale != 0.0, "scale should be non-zero in Rescaler."
 
     def rescale_candidate(self, candidate: p.Parameter, inverse: bool = False) -> p.Parameter:
         data = candidate.get_standardized_data(reference=self.parametrization)
         scale = self.scale if not inverse else 1.0 / self.scale
-        return self.parametrization.spawn_child().set_standardized_data(scale * data)
+        return self.parametrization.spawn_child().set_standardized_data(scale * data + self.initialization)
```

Comment:
note that there are two cases: inverse=True or False, and they must be dealt with for the tell not asked to work